### PR TITLE
Display the obstacles of the routes

### PR DESCRIPTION
**Pull request by tuukka**

### CODE REVIEW

File: src/planner.ts [hunk 1]
```diff
@@ -4,11 +4,28 @@ import { FeatureCollection } from "geojson";
...
+      // eslint-disable-next-line no-console
+      console.log(
+        step.through,
+        node.definedTags[
+          "https://w3id.org/openstreetmap/terms#barrier"
+        ].replace(/^.*#/, ""),
+        node.id,
+        node.definedTags,
+        node.freeformTags
+      );
```
Comment (haphut) [inline src/planner.ts:53]:
```suggestion
```

Comment (haphut) [inline src/planner.ts:53]:
```suggestion
```

File: src/planner.ts [hunk 2]
```diff
@@ -90,11 +118,14 @@ export default function calculatePlan(
           // eslint-disable-next-line @typescript-eslint/no-explicit-any
           .on("data", async (path: any) => {
             const completePath = await planner.completePath(path);
-            const geometry = extractGeometry(completePath);
+            // eslint-disable-next-line no-console
+            console.log(completePath);
```
Comment (haphut) [inline src/planner.ts]:
```suggestion
```

File: src/planner.ts (hunk 1 above)
Comment (tuukka) [reply src/planner.ts:53]: Will be handled under another issue

**Pull request merged**